src/components/Classroom.tsx

```diff
@@ -0,0 +1,12 @@
+import * as React from 'react'
+
+type ClassroomProps = {
+  key: string
```

Comment:
Unclear if needed per discussion

Reply:
@DoctorBurglar  did you resolve this?  we should merge this PR soon, it's got 11 commits in it